src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machinesArray = {new Truck(), new Bulldozer(), new Excavator()};
 
```

Comment:
```suggestion
```

Reply:
Added fixes after review. Could you check it, please?